storybook/__snapshots__/storyshots.test.js.snap

```diff
@@ -47778,7 +47778,7 @@ exports[`Storyshots TextInput default 1`] = `
                     "borderTopWidth": 0,
                     "borderWidth": undefined,
                     "flexDirection": "row",
-                    "height": 54,
+                    "height": 62,
```

Comment:
This change seems to cause input value to break into two lines. AFAIK it should stay in one line, pls consult with Dmitry if necessary.


![image](https://user-images.githubusercontent.com/6368606/116249537-3658df80-a76d-11eb-85b9-a951b3f667fd.png)

Reply:
This was as per the discussion with Dmitry @paragramm. Dmitry can you please confirm.

Reply:
@paragramm could you please confirm this please mate?